CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+Coming soon
```

Comment:
Go ahead and add the actual content for `contributing`

You can view an example at https://github.com/okta/okta-sdk-golang/blob/master/CONTRIBUTING.md This is the one that we have kind of landed on for all our repos.

Reply:
Done